CVLByExample/summary/with-side-effects/README.md

```diff
@@ -0,0 +1,37 @@
+# Summarization for Multi Contracts with Side Effects
+
```

Comment:
the focus here should be dispatcher and link

Reply:
done

Reply:
done for function summary. ghost summary still vacuous